msgid ""
msgstr "Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-04-08 15:01+0200\n"
"PO-Revision-Date: 2014-04-09 05:39-0400\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Hungarian\n"
"Language: hu_HU\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.net\n"

#: pyload.py:182
msgid "Received Quit signal"
msgstr "Leállítási jel érkezett"

#: pyload.py:303
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "a pyLoad már fut az alábbi pid számmal: %s"

#: pyload.py:317
#, python-format
msgid "Failed changing group: %s"
msgstr "Sikertelen csoport váltás: %s"

#: pyload.py:327
#, python-format
msgid "Failed changing user: %s"
msgstr "Sikertelen felhasználó váltás: %s"

#: pyload.py:329
msgid "folder for logs"
msgstr "könyvtár a naplóknak"

#: pyload.py:340
msgid "Starting"
msgstr "Indítás"

#: pyload.py:341
#, python-format
msgid "Using home directory: %s"
msgstr "Főkönyvtár használata: %s"

#: pyload.py:350
msgid "pycrypto to decode container files"
msgstr "pycrypto a konténer fájlok dekódolásához"

#: pyload.py:353
msgid "folder for temporary files"
msgstr "könyvtár az átmeneti fájloknak"

#: pyload.py:358
msgid "folder for downloads"
msgstr "könyvtár a letöltéseknek"

#: pyload.py:361
msgid "OpenSSL for secure connection"
msgstr "OpenSSL a biztonságos kapcsolathoz"

#: pyload.py:365
msgid "Moving old user config to DB"
msgstr ""

#: pyload.py:368
msgid "Please check your logindata with ./pyload.py -u"
msgstr "Ellenőrizd a belépési adataid a következő paranccsal: ./pyload.py -u"

#: pyload.py:371
msgid "All links removed"
msgstr "Minden link törölve"

#: pyload.py:402
#, python-format
msgid "Downloadtime: %s"
msgstr "Letöltési idő: %s"

#: pyload.py:412
#, python-format
msgid "Free space: %s"
msgstr "Szabad hely: %s"

#: pyload.py:432
msgid "Activating Accounts..."
msgstr "Hozzáférések aktiválása..."

#: pyload.py:438
msgid "Activating Plugins..."
msgstr "Bővítmények aktiválása..."

#: pyload.py:441
msgid "pyLoad is up and running"
msgstr "pyLoad fut"

#: pyload.py:460
msgid "restarting pyLoad"
msgstr "pyLoad újraindítása"

#: pyload.py:464
msgid "pyLoad quits"
msgstr "pyLoad kilép"

#: pyload.py:521
#, python-format
msgid "Install %s"
msgstr "Telepítés %s"

#: pyload.py:557
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr ""

#: pyload.py:559
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr ""

#: pyload.py:580
msgid "shutting down..."
msgstr "leállítás..."

#: pyload.py:597
msgid "error while shutting down"
msgstr "hiba leállítás közben"

#: pyload.py:661
msgid "killed pyLoad from Terminal"
msgstr "pyLoad kilőve konzolról"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr ""

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr ""

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr ""

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr ""

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr ""

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "befejezett"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr ""

#: module/database/FileDatabase.py:45
msgid "online"
msgstr ""

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "várólistán"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "átugorva"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "várakozás"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "átmenetileg offline"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "indítás"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "sikertelen"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "megszakított"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "dekódolás"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "egyéni"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "letöltés"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "feldolgozás alatt"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "ismeretlen"

#: module/database/FileDatabase.py:531 module/plugins/hooks/IRCInterface.py:74
#: module/plugins/hooks/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Kész csomag: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "SSL ThriftBackend használata"

#: module/manager/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Távoli hiba: %s"

#: module/manager/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Indítás: %(name)s: %(addr)s:%(port)s"

#: module/manager/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Backend betöltése sikertelen: %(name)s | %(error)s"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "várakozás %s"

#: module/web/ServerThread.py:35
msgid "SSL certificates not found."
msgstr "SSL tanúsítvány nem található."

#: module/web/ServerThread.py:39
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr ""

#: module/web/ServerThread.py:40
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr ""

#: module/web/ServerThread.py:41
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr ""

#: module/web/ServerThread.py:42
msgid "sample configs are located in the module/web/servers directory"
msgstr ""

#: module/web/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr ""

#: module/web/ServerThread.py:56
#, python-format
msgid "Error importing lightweight server: %s"
msgstr ""

#: module/web/ServerThread.py:57
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr ""

#: module/web/ServerThread.py:58
msgid "Copy the boern.so to module/lib folder or use setup.py install"
msgstr ""

#: module/web/ServerThread.py:59
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr ""

#: module/web/ServerThread.py:63
msgid "Server set to threaded, due to known performance problems on windows."
msgstr ""

#: module/web/ServerThread.py:80 module/web/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Ez a szerver nem támogatja az SSL-t, kérlek használj helyette másikat"

#: module/web/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr ""

#: module/web/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr ""

#: module/web/pyload_app.py:125
msgid "You dont have permission to access this page."
msgstr "Nincs jogosultságod a lap megtekintéséhez."

#: module/web/pyload_app.py:193
msgid "Download directory not found."
msgstr "Letöltési könyvtár nem található."

#: module/web/pyload_app.py:260 module/web/pyload_app.py:267
msgid "unlimited"
msgstr "végtelen"

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "not available"
msgstr "nem elérhető"

#: module/web/pyload_app.py:508
msgid "Run pyload.py -s to access the setup."
msgstr "A beállításhoz futtasd: pyload.py -s."

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Darabokban letöltés sikertelen, visszatérés egyszeri kapcsolatra | %s"

#: module/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Letöltés elindítva: %s"

#: module/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Letöltés befejezve: %s"

#: module/PluginThread.py:194 module/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "A %s bővítményből hiányzik egy funkció."

#: module/PluginThread.py:202 module/PluginThread.py:265
#: module/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Letöltés megszakítva: %s"

#: module/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Letöltés újraindítva: %(name)s | %(msg)s"

#: module/PluginThread.py:231 module/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "A letöltés offline: %s"

#: module/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "A letöltés átmenetileg offline: %s"

#: module/PluginThread.py:237 module/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "A letöltés sikertelen: %(name)s | %(msg)s"

#: module/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "Nem sikerült csatlakozni a címhez, újrapróbálás 1 perc múlva."

#: module/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Letöltés átugorva: %(name)s - %(plugin)s"

#: module/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Dekódolás elkezdve: %s"

#: module/PluginThread.py:376 module/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Dekódolás sikertelen: %(name)s | %(msg)s"

#: module/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Újra próbálkozás %s"

#: module/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "%(name)s információinak lekérése sikertelen | %(err)s"

#: module/manager/HookManager.py:90 module/plugins/Hook.py:102
#, python-format
msgid "Error executing hooks: %s"
msgstr ""

#: module/manager/HookManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Sikertelen aktiválás: %(name)s"

#: module/manager/HookManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr ""

#: module/manager/HookManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr ""

#: module/manager/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Újracsatlakozás sikertelen: %s"

#: module/manager/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Újracsatlakozó script nem található!"

#: module/manager/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Újracsatlakozás indítása"

#: module/manager/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Az újracsatlakozó scriptet nem sikerült futtatni!"

#: module/manager/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Újracsatlakozva, új IP: %s"

#: module/manager/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Nincs elég hely az eszközön"

#: module/plugins/Account.py:85 module/plugins/Account.py:91
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Nem sikerült bejelentkezni: %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Hibás jelszó"

#: module/plugins/Account.py:240
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr ""

#: module/plugins/Account.py:266
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "%s hozzáférésen nincs elég adatforgalom, újra próbálás: 30 perc"

#: module/plugins/Account.py:273
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "%s hozzáférés lejárt, újra próbálás: 1 óra"

#: module/plugins/crypter/SerienjunkiesOrg.py:128
msgid "Downloadlimit reached"
msgstr "A letöltési korlát elérve"

#: module/manager/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr ""

#: module/manager/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Hiba a(z) %(name)s betöltése közben: %(msg)s"

#: module/plugins/internal/MultiHoster.py:133
msgid "No Hoster loaded"
msgstr "Nincs szolgáltató betöltve"

#: module/plugins/accounts/BitshareCom.py:37
msgid "Activate direct Download in your Bitshare Account"
msgstr "Közvetlen letöltés aktiválása a Bitshare hozzáférésen"

#: module/plugins/container/LinkList.py:65
msgid "LinkList could not be cleared."
msgstr ""

#: module/manager/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr ""

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Azonosítás szükséges (felhasználónév:jelszó)"

#: module/plugins/hoster/SimplydebridCom.py:25
#: module/plugins/hoster/RealdebridCom.py:41
#: module/plugins/hoster/FreeWayMe.py:38 module/plugins/hoster/ZeveraCom.py:22
#: module/plugins/hoster/UnrestrictLi.py:53
#: module/plugins/hoster/Premium4Me.py:28 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/DebridItaliaCom.py:40
#: module/plugins/hoster/RPNetBiz.py:26
#: module/plugins/hoster/MultiDebridCom.py:41
#: module/plugins/hoster/ReloadCc.py:24 module/plugins/hoster/RehostTo.py:26
#: module/plugins/hoster/PremiumizeMe.py:22
#: module/plugins/hooks/RPNetBiz.py:43
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Add meg a(z) %s hozzáférés adatait, vagy deaktiváld ezt a beépülőt"

#: module/plugins/hoster/FilesMailRu.py:99
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "HTML kód van a letöltött fájlban (%s). Lehet, hogy átirányítási hiba, a letöltés újra lesz indítva."

#: module/plugins/hoster/NetloadIn.py:146
#: module/plugins/hoster/NetloadIn.py:170
msgid "File temporarily not available"
msgstr "Állomány átmenetileg nem elérhető"

#: module/plugins/hoster/NetloadIn.py:183
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: letöltések közti várakozás %d mp."

#: module/plugins/hoster/NetloadIn.py:214
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: captcha-ra várakozás %d mp."

#: module/plugins/hoster/NetloadIn.py:252
msgid "Downloaded File was empty"
msgstr "A letöltött fájl üres"

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "Érvénytelen API kulcs"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Nincs elég adatforgalom hátra"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Adatforgalom túllépve"

#: module/plugins/hoster/RapidshareCom.py:100
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Adatforgalom  megosztás (közvetlen letöltés)"

#: module/plugins/hoster/RapidshareCom.py:127
#: module/plugins/hoster/RapidshareCom.py:194
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Már van folyamatban letöltés erről az IP-ről, várakozás: 60mp"

#: module/plugins/hoster/RapidshareCom.py:131
msgid "Invalid Auth Code, download will be restarted"
msgstr "Hibás Auth kód, a letöltés újraindítása"

#: module/plugins/hoster/RapidshareCom.py:199
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: Nincs ingyenes hely"

#: module/plugins/hoster/RapidshareCom.py:202
msgid "You need a premium account for this file"
msgstr "Ehhez a fájlhoz prémium hozzáférés szükséges"

#: module/plugins/hoster/RapidshareCom.py:204
msgid "Filename reported invalid"
msgstr "Hibás fájlnév"

#: module/plugins/hoster/FileserveCom.py:99
msgid "Parallel download error, now waiting 60s."
msgstr "Többszálas letöltés hiba, várakozás: 60s."

#: module/plugins/hoster/FileserveCom.py:215
msgid "Not logged in."
msgstr "Nem vagy bejelentkezve."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Sikertelen dekódolás"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Nincs fájl kulcs az URL-ben"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Hibakód:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr ""

#: module/plugins/hooks/UpdateManager.py:74
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr "*** Bővítmények frissítve, indítsd újra a pyLoad-ot ***"

#: module/plugins/hooks/UpdateManager.py:76
msgid "Plugins updated and reloaded"
msgstr "Bővítmények frissítve és betöltve"

#: module/plugins/hooks/UpdateManager.py:79
msgid "No plugin updates available"
msgstr "Nincs bővítmény frissítés"

#: module/plugins/hooks/UpdateManager.py:96
msgid "No Updates for pyLoad"
msgstr "Nincs új pyLoad verzió"

#: module/plugins/hooks/UpdateManager.py:100
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Új pyLoad verzió elérhető: %s  ***"

#: module/plugins/hooks/UpdateManager.py:101
msgid "***  Get it here: http://pyload.org/download  ***"
msgstr "***  Letöltés: http://pyload.org/download  ***"

#: module/plugins/hooks/UpdateManager.py:104
msgid "Not able to connect server for updates"
msgstr "Nem sikerült a szerverhez csatlakozni a frissítésekért"

#: module/plugins/hooks/UpdateManager.py:148
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr "Új verzió: %(type)s|%(name)s : %(version).2f"

#: module/plugins/hooks/UpdateManager.py:157
#: module/plugins/hooks/UpdateManager.py:162
#, python-format
msgid "Error when updating %s"
msgstr "Hiba frissítés közben %s"

#: module/plugins/hooks/UpdateManager.py:162
msgid "Version mismatch"
msgstr "verzió ütközés"

#: module/plugins/hooks/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Letöltés befejezve: %(name)s @ %(plugin)s "

#: module/plugins/hooks/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Új captcha kérés: %s"

#: module/plugins/hooks/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Válaszolj ezzel: 'c %s szöveg a captcha-n'"

#: module/plugins/hooks/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Először adj meg rehost.to hozzáférést majd indítsd újra a pyLoad-ot"

#: module/plugins/hooks/HotFolder.py:82
#, python-format
msgid "Added %s from HotFolder"
msgstr "%s HotFolder-ből hozzáadva"

#: module/plugins/hooks/ExtractArchive.py:95
#, python-format
msgid "No %s installed"
msgstr "Nincs %s telepítve"

#: module/plugins/hooks/ExtractArchive.py:97
#: module/plugins/hooks/ExtractArchive.py:102
#, python-format
msgid "Could not activate %s"
msgstr "Nem sikerült aktiválni: %s"

#: module/plugins/hooks/ExtractArchive.py:107
msgid "Activated"
msgstr "Aktivált"

#: module/plugins/hooks/ExtractArchive.py:109
msgid "No Extract plugins activated"
msgstr "Nincs kicsomagoló bővítmény aktiválva"

#: module/plugins/hooks/ExtractArchive.py:121
#, python-format
msgid "Package %s queued for later extracting"
msgstr "%s csomag a várólistán későbbi kitömörítésre"

#: module/plugins/hooks/ExtractArchive.py:144
#, python-format
msgid "Check package %s"
msgstr "Csomag ellenőrzése: %s"

#: module/plugins/hooks/ExtractArchive.py:185
#, python-format
msgid "Extract to %s"
msgstr "Kicsomagolás: %s"

#: module/plugins/hooks/ExtractArchive.py:200
msgid "No files found to extract"
msgstr "Nincs kitömöríthető fájl"

#: module/plugins/hooks/ExtractArchive.py:207
msgid "extracting"
msgstr "kicsomagolás"

#: module/plugins/hooks/ExtractArchive.py:218
msgid "Password protected"
msgstr "Jelszóval védett"

#: module/plugins/hooks/ExtractArchive.py:239
msgid "Wrong password"
msgstr "Hibás jelszó"

#: module/plugins/hooks/ExtractArchive.py:247
#, python-format
msgid "Deleting %s files"
msgstr "%s fájl törlése"

#: module/plugins/hooks/ExtractArchive.py:254
msgid "Extracting finished"
msgstr "Kicsomagolás kész"

#: module/plugins/hooks/ExtractArchive.py:260
msgid "Archive Error"
msgstr "Tömörített állomány hiba"

#: module/plugins/hooks/ExtractArchive.py:262
msgid "CRC Mismatch"
msgstr "CRC hiba"

#: module/plugins/hooks/ExtractArchive.py:266
msgid "Unknown Error"
msgstr "Ismeretlen hiba"

#: module/plugins/hooks/ExtractArchive.py:318
msgid "Setting User and Group failed"
msgstr "Felhasználó és csoport beállítás sikertelen"

#: module/plugins/hooks/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: A 9666 port használatban van"

#: module/plugins/hooks/CaptchaTrader.py:69
#: module/plugins/hooks/Captcha9kw.py:59
#: module/plugins/hooks/ExpertDecoders.py:50
#, python-format
msgid "%s credits left"
msgstr "%s kredit maradt"

#: module/plugins/hooks/CaptchaTrader.py:117
msgid "Could not send response."
msgstr "Nem sikerült választ küldeni."

#: module/plugins/hooks/CaptchaTrader.py:135
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "A CaptchaTrader hozzáférésen nincs elég credit"

#: module/plugins/hooks/LinkdecrypterCom.py:43
msgid "Crypter list not found"
msgstr "Crypter lista nem található"

#: module/plugins/hooks/LinkdecrypterCom.py:57
msgid "Crypter list is empty"
msgstr "Crypter lista üres"

#: module/plugins/hooks/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Letöltés befejezve: %(name)s @ %(plugin)s"

#: module/plugins/hooks/Captcha9kw.py:93
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Új CaptchaID a feltöltésből:  %s : %s"

#: module/plugins/hooks/Captcha9kw.py:129
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "A Captcha 9kw.eu hozzáférésen nincs elég credit"

#: module/plugins/hooks/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Scriptek telepítve - %s: "

#: module/plugins/hooks/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Script nem futtatható:"

#: module/plugins/hooks/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Hiba: %(script)s: %(error)s"

#: module/plugins/hooks/ExpertDecoders.py:95
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Az ExpertDecoders hozzáférésen nincs elég credit"

#: module/plugins/hooks/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Először adj meg rehost.to hozzáférést majd indítsd újra a pyLoad-ot."

#: module/plugins/hooks/PremiumizeMe.py:48
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Először adj meg premiumize.me hozzáférést majd indítsd újra a pyLoad-ot."

#: module/plugins/hooks/CaptchaBrotherhood.py:69
#, python-format
msgid "%d credits left"
msgstr "%d kredit maradt"

#: module/plugins/Plugin.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Captcha megfejtéshez nincs kliens csatlakozva és nincs telepítve Pil vagy tesseract"

#: module/plugins/Plugin.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr ""

#: module/plugins/Plugin.py:498 module/plugins/Plugin.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Felhasználó és csoport beállítás sikertelen: %s"

#: module/manager/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr ""

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr ""

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr ""

#: module/utils/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr ""

